ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -366,20 +371,22 @@ const AccountListItem = ({
         </Box>
         {accountLabels.length > 0 ? (
           <Box flexDirection={FlexDirection.Row}>
-            {accountLabels.map(({ label, icon }) => {
-              return (
-                <Tag
-                  data-testid={`account-list-item-tag-${account.id}-${label}`}
-                  key={label}
-                  label={label}
-                  labelProps={{
-                    variant: TextVariant.bodyXs,
-                    color: Color.textAlternative,
-                  }}
-                  startIconName={icon}
-                />
-              );
-            })}
+            {accountLabels
+              .filter(({ label }) => !(isSrpPill(label) && !showSrpPill))
```

Comment:
Do we need this second loop (the filter)?  Can we just return null in the `map` to cut down on double-looping?